lib/lambda-cfn.js

```diff
@@ -381,6 +381,13 @@ function lambda(options) {
     fn.Properties.MemorySize = 128;
   }
 
+  if (options.runtime) {
```

Comment:
@alulsh what happens if options.runtime is not specified?  Will Runtime get set to "" and if so, is that valid?  A test around this may make sense.